src/functions/wceStyles.ts

```diff
@@ -367,11 +367,11 @@ export default function wceStyles(): void {
   }
   #wce-chat-baby-talk[data-state="ignore"]::before,
   #wce-chat-stutters[data-state="ignore"]::before {
-    background-color: #bbb;
+    --button-color: #bbb;
   }
   #wce-chat-baby-talk[data-state="preserve"]::before,
   #wce-chat-stutters[data-state="preserve"]::before {
-    background-color: #666;
+    --button-color: #666;
```

Comment:
same

```suggestion
    ${GameVersion === "R104" ? "background" : "--button"}-color: #666;
```

Reply:
Applied as of https://github.com/KittenApps/WCE/pull/29/commits/7125374c0e755734d7e55e86ceacb3829d20d5b5